tests/API/XCCDF/unittests/test_generate_fix_ansible_vars_golden.yml

```diff
@@ -0,0 +1,18 @@
+---
+ - hosts: all
+   vars:
+      www_value_val1: 100
```

Comment:
I think it makes sense to have a test with multiples variables as well, at least two.

Reply:
Also, it would be nice to check variables in playbooks generated from tailored Profiles.